nixos/tests/onlyoffice.nix

```diff
@@ -0,0 +1,29 @@
+let
+  port = 8000;
+in
+{
+  name = "onlyoffice";
+
+  nodes.machine =
+    { pkgs, ... }:
+    {
+      services.onlyoffice = {
+        enable = true;
+        inherit port;
+        hostname = "office.example.com";
+        securityNonceFile = "${pkgs.writeText "nixos-test-onlyoffice-nonce.conf" ''
+          set $secure_link_secret "nixostest";
+        ''}";
+      };
+
+      networking.hosts = {
+        "::1" = [ "office.example.com" ];
+      };
+    };
+
+  testScript = ''
+    machine.wait_for_unit("onlyoffice-docservice.service")
```

Comment:
maybe also check that `onlyoffice-converter.service` successfully started?

Reply:
yeah, will add that - sorry I am still executing this test locally, please wait with merging ;)